src/somesy/core/writer.py

```diff
@@ -150,7 +150,11 @@ def _merge_person_metadata(old: List[Person], new: List[Person]) -> List[Person]
                 # if there were changes -> update person
                 overlapping_fields = person.dict(include=set(person_update.keys()))
                 if person_update != overlapping_fields:
+                    print("updating ", modified_people[i], "->", person_update)
```

Comment:
I think we should log these.

Reply:
I forgot to remove them. Do you think we should actually debug log stuff in here anyway? I'm thinking it could be a bit too verbose.

Reply:
Since there might be a change to the outputs, I thought it would be nice to see the changes

Reply:
Done